Exercises/NoModules/Chapter 02/Soln2_07.cpp

```diff
@@ -0,0 +1,23 @@
+// Exercise 2-7. Fun with formatting.
```

Comment:
I don't have this exercise in my draft version of the text.
But it's compiling and running fine.

Reply:
Strange. It is in the final text:

"Add a row to your table of Exercise 2-6 for sin(π/4), showing the result with
exponent notation and five digits after the decimal point. Make sure the exponent component
begins with a capital E, not a lowercase e."